opennms-doc/guide-admin/src/asciidoc/text/telemetryd/protocols/netflow5-adapter.adoc

```diff
@@ -0,0 +1,21 @@
+
+[[telemetryd-netflow5-adapter]]
+===== NetFlow v5 Adapter
+
+The NetFlow v5 adapter is used to handle _NetFlow v5_ payloads.
+
+====== Facts
+
+[options="autowidth"]
+|===
+| Class Name          | `org.opennms.netmgt.telemetry.adapters.netflow.Netflow5Adapter`
+|===
+
+====== Parameters
+
+.Adapter specific parameters for the Netflow5Adapter
+[options="header, autowidth"]
+|===
+| Parameter        | Description                                                       | Required | Default value
+| `script`         | Full path to the script used to handle the JTI messages           | required | (none)
```

Comment:
The Netflow adapter does not use a script. There are no parameters as far as I know.